README.md

```diff
@@ -10,7 +10,7 @@ To see it in action, we have made example:
 
 1. Sinatra, which demonstrate in as succint code as possible. Please [have a look here](https://github.com/veritrans/veritrans-ruby/tree/master/example/sinatra)
```

Comment:
<img width="288" alt="Screen Shot 2021-09-30 at 17 12 04" src="https://user-images.githubusercontent.com/13027142/135439617-9f527e6a-a08d-4c56-a959-0f279b37f865.png">
Readme structure is a mess, why is section 2 level is under 1? it should be on the same level.

So many word still using Veritrans, revamp it properly to Midtrans. Also still using vt-web, rename it to proper snap redirect, etc. I expect you to revamp the readme too, make it up to date with current situation, read them all and carefully revamp them all. Not just few part. Use other library's readme as a benchmark, especially NodeJS or Go, which is the most recently updated.

Change the overall structure and the content if needed, I expect you to make it up to date and clearer for merchant to understand it easily.

Reply:
I expect you to think & take initiative in improving the readme, not just waiting for me to give order which part to change, no one want to be robot and just take orders.

Reply:
This library readme should be at least as good as Go or NodeJS library, or better. That's the standard. Currently the readme is a mess and outdated.

Reply:
The Core API part is also a mess, the sample HTML is not enough for card transaction, there is no guide (or link) on how to open 3DS, handle the JS callback etc.

Reply:
i'm so sorry kak, I'm working in a hurry because I think I have to finish the revision today. In the future I will be more careful and not in a hurry
Thanks a lot ka